src/bun.js/event_loop.zig

```diff
@@ -481,7 +481,7 @@ pub const Task = TaggedPointerUnion(.{
     TimerObject,
     bun.shell.Interpreter.Builtin.Yes.YesTask,
 
-    bun.kit.DevServer.BundleTask,
+    // bun.kit.DevServer.BundleTask,
```

Comment:
can this be deleted if it's not used?

Reply:
yes. this structure has been fully deleted